code/datums/spells/horsemask.dm

```diff
@@ -26,7 +26,8 @@
 	if(!target)
 		return
 
-	if((target.type in compatible_mobs) || ishuman(target))
+
+	if(!is_type_in_list(target, compatible_mobs))

```

Comment:
I don't see what harm comes from checking compatible mobs, on the up side if you ever want to make this spell for other species you have less code to edit.